.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+name: Kmesh CI Workflow
+
+on:
+  push:
+    branches: [ "main"]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        go-version: [ '1.21' ]
+    
+    env:
+      GOPATH: /home/runner/go
+      GOBIN: /home/runner/go/bin
+      PKG_CONFIG_PATH: $GITHUB_WORKSPACE/mk
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Go
+      uses: actions/setup-go@v4.0.0
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Install Go Dependence
+      run: |
+        go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32
+        sudo cp $GOBIN/* /usr/bin
```

Comment:
we should put it in build.sh. FYI https://github.com/kmesh-net/kmesh/pull/88/files#r1453054773

Reply:
We will remove this step, then the PR should be merged after merging PR #88.

Reply:
> we should put it in build.sh. FYI https://github.com/kmesh-net/kmesh/pull/88/files#r1453054773

We will remove this step, then the PR should be merged after merging PR #88.

Reply:
We will remove this step, then the PR should be merged after merging PR #88.